### PR TITLE
Create a pedia article for Technologies

### PR DESCRIPTION
**Pull request by Ouaz**
- Explain briefly how works a technology unlock
- Point out what are the required conditions for a ship to receive a part upgrade
- Fix some typos
- Add some pedia links

### CODE REVIEW

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -4355,7 +4355,7 @@ RESEARCH_TEXT
 '''The Research (or RP "Research Points") of all planets in the empire is combined to discover new technologies. Any Research that cannot be spent on a tech is lost each turn. Each tech has a minimum number of turns required to research it. For example, if a tech that cost 15 RPs, and had a minimum time of 3 turns, 5 RPs is the maximum that could be put towards its completion per turn.
 The tech at the top of the queue is given RPs first. Any remaining RP are then given to the next tech in the queue, and if there is still more remaining to the next-- and so on. You can re-prioritize the items in the queue by dragging the most important items closer to the top.
 
-Research does not require [[encyclopedia SUPPLY_TITLE]] line connections to be used.'''
+Research does not require [[encyclopedia SUPPLY_TITLE]] line connections to be used. However, if a tech which improves a ship part is researched (e.g. a weapon), the already existing ships must be in supply range, stationary, and not in combat, in order to receive the upgrade during the following turn.'''
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: This addition doesn't make sense here. The article is about how the queue works and how RP are spent, not about how techs affect ship parts.

**Review by geoffthemedio — CHANGES_REQUESTED** [review]

File: default/stringtables/en.txt (hunk 1 above)
Comment (Ouaz) [reply default/stringtables/en.txt]: Well,  for the most part,  Tech propagation does not require supply line connections.

So, it's easy to think that everything related to Research and Techs does not need any supply line connections.
Except the ship part improvements for the already existing ships.

But I didn't know where to specify it, elsewhere than in the Research article.

Maybe reword it?

`Note: for the most part, an item unlocked by a newly researched tech does not require supply line connections to be available or active (e.g. a building or a planetary meter improvement). However, if a tech which improves a ship part is researched [...]`

Comment (dbenage-cx) [reply default/stringtables/en.txt]: The research article is linked from ~30 stringtable entries.  If not dropped, I'd opt for a new paragraph here with a more general wording (ship parts and supply lines are not the only requirement pairs).

Something like:

`Research that unlocks an item will only give an empire the knowledge of how to construct that item.  Actually constructing such an item may have other requirements, such as access to a certain resource.`
`If an item has been unlocked and is not available, check the items pedia entry for requirements.`
`Another option is to check the hover text on the items listing in the production window (with Unavailable selected).`

Comment (geoffthemedio) [reply default/stringtables/en.txt]: This would make more sense as a new article titled "Technologies" or similar. The article being modified here is about the research output of planets, not what effects techs have once completed.

**Comment by Ouaz** [comment]
#
EDIT: oh, I just thought... maybe a simple beginner hint would do the trick?
#

I understand what means Geoff.

But as pointed out by dbenage-cx, there's a lot of Research pedia link in the stringtables. Create a lonely article about how works a tech unlock would not be very useful (a new player would have to dig in the pedia himself).

By the way, I made this PR because of this topic in the forum: 
http://www.freeorion.org/forum/viewtopic.php?p=85478#p85478

What about a sub-paragraph in the Research article then?

Something like:

```
[u]Note about technology unlocks[/u]

Research that unlocks an item will only give an empire the knowledge of how to construct that item.
 Actually constructing such an item may have other requirements, such as access to a certain resource.
If an item has been unlocked and is not available, check the items pedia entry for requirements.
Another option is to check the hover text on the items listing in the production window (with Unavailable selected).

etc.
```

About the improved ship part and the required resupply in order the already existing ships to be upgraded , it is not obvious it works like this (although it is a clever gameplay mechanic, more realistic).

Generally, in video games, when you unlock an improvement, all your existing stuff is directly upgraded (it's a kind of magic ^^).

That's why It wanted to specify it somewhere in the pedia (and easily reachable by a beginner).

**Comment by dbenage-cx** [comment]
A little less noticeable, but a link in Research to Technology would be easy to introduce.
Some of those existing links may then be updated, if they meant to refer to techs.

**Comment by Ouaz** [comment]
Hmmm... The new PR Review feature does not seem to tag the review as outdated (or "outdated diff", as usual) when the PR content is modified...

Whatever, as requested, I created a dedicated article for "Technologies" in "Game Concepts" category, and linked it in the Research article.

As usual, maybe it needs some rewording. ^^

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -4352,11 +4352,25 @@ RESEARCH_TITLE
...
+Any Research that cannot be spent on a tech is lost each turn. Each tech has a minimum number of turns required to research it. For example, if a tech that cost 15 RPs, and had a minimum time of 3 turns, 5 RPs is the maximum that could be put towards its completion per turn.
 The tech at the top of the queue is given RPs first. Any remaining RP are then given to the next tech in the queue, and if there is still more remaining to the next-- and so on. You can re-prioritize the items in the queue by dragging the most important items closer to the top.
 
 Research does not require [[encyclopedia SUPPLY_TITLE]] line connections to be used.'''
 
+TECHNOLOGIES_TITLE
+Technologies
+
+TECHNOLOGIES_TEXT
+'''New attributes, improved planetary stats, buildings, and ship parts can be unlocked by researching the associated technologies.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: What's an attribute?

File: default/stringtables/en.txt [hunk 3]
```diff
@@ -4352,11 +4352,25 @@ RESEARCH_TITLE
...
+
+[[encyclopedia RESEARCH_TITLE]] a technology that unlocks an item will only give an empire the knowledge of how to construct that item. Actually constructing such an item may have other requirements, such as access to a certain resource.
+If an item has been unlocked and is not available, the requirements can be checked in the item pedia entry.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: Doesn't have to be unlocked or not available to check such, does it?

File: default/stringtables/en.txt [hunk 4]
```diff
@@ -4352,11 +4352,25 @@ RESEARCH_TITLE
...
+Another option is to check the hover text on the items listing in the Production window (with [[PRODUCTION_WND_AVAILABILITY_UNAVAILABLE]] selected).
+
+For the most part, an item unlocked by a newly researched technology does not require [[encyclopedia SUPPLY_TITLE]] line connections to be available or active (e.g. a building or a planetary meter improvement). However, if a tech which improves a ship part is researched (e.g. a weapon), the already existing ships must be in supply range, stationary, and not in combat, in order to receive the upgrade during the following turn.'''
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: Avoid florid phrases like "For the most part". Prefer "Generally" or "Normally"

Comment (geoffthemedio) [inline default/stringtables/en.txt]: already-existing

File: default/stringtables/en.txt (hunk 2 above)
Comment (Ouaz) [reply default/stringtables/en.txt]: I didn't know how to name it... For example, it's the 10% repair bonus granted by techs like Advanced Damage Control.

Comment (Ouaz) [reply default/stringtables/en.txt]: OK, "effects" is much better. :)

**Comment by Ouaz** [comment]
All requested changes made.

But this new review system is definitely bad when a commit is amended then force-pushed, as it doesn't mark the previous comments as outdated.

Makes the comments section very messy now, IMO.
